.github/workflows/maintenance.yml

```diff
@@ -0,0 +1,32 @@
+name: Weekly maintenance/update tasks
+on:
+    workflow_dispatch:
+    schedule:
+        - cron: "30 1 * * 1" # Run on Mondays @ 7:30am UTC (3:30am EDT, 12:30 am PDT)
```

Comment:
This field is compatible with POSIX `cron` syntax. More details here: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule

Reply:
**suggestion:** For easier workflow maintenance, I usually paste this useful info into the workflow as a comment...like this:
https://github.com/Khan/webapp/blob/906784c2673fa8fcd7e1a6aced02601692d646d7/.github/workflows/static-service-nightly-e2e.yml#L7-L15